### PR TITLE
[faucet] Improve logging

### PR DESCRIPTION
**Pull request by mcortesi**
### Description

Log rpc ins and outs. Plus other logs.

Removes unused code

### Tested

deployed, went to faucet web, and tried it

### CODE REVIEW

File: packages/faucet/scripts/test-celo-adapter.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import Web3 from 'web3'
+import { CeloAdapter } from '../src/celo-adapter'
+import { getAddress } from '../src/tx'
+
+// Obtained from `yarn cli config:get --alfajores
```
Comment (m-chrzan) [inline packages/faucet/scripts/test-celo-adapter.ts]: Typo? Shouldn't this be `... --net alfajores`? Also, missing `` ` `` at the end

File: packages/faucet/src/tx.ts [hunk 2]
```diff
@@ -96,19 +67,20 @@ export class TransactionResult {
...
+      console.info('@TransactionResult hash: ', hash)
       this.hashFuture.resolve(hash)
     })
       .on('receipt', (receipt: TransactionReceipt) => {
-        debugTxObjects('receipt: %O', receipt)
+        console.info('@TransactionResult receipt: ', receipt)
         this.receiptFuture.resolve(receipt)
       })
       .on('error', ((error: any, receipt: TransactionReceipt | false) => {
+        console.info('got a future error', error, receipt)
```
Comment (m-chrzan) [inline packages/faucet/src/tx.ts]: This error message can probably be reworded. `'@TransactionResult error:` to conform with the other ones? And maybe only log the receipt in the second branch of the `if` below.

**Review by m-chrzan — DISMISSED** [review]
Just one nit about wording.

**Review by m-chrzan — APPROVED** [review]

**Comment by mcortesi** [comment]
@timmoreton @asaj do you have permission? can you merge?

**Comment by mcortesi** [comment]
let me know so i can cherry pick to master

**Pull request merged**